### PR TITLE
F34032; Accessibility Feature

### PR DESCRIPTION
**Pull request by byersc1122**

### CODE REVIEW

File: collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml [hunk 1]
```diff
@@ -113,7 +112,7 @@
         <RelativeLayout
             android:id="@+id/frame_filter_icon"
             android:layout_width="wrap_content"
-            android:layout_height="match_parent"
+            android:layout_height="0dp"
```
Comment (jjchan1) [inline collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml:115]: What is this supposed to do?

File: collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml [hunk 2]
```diff
@@ -138,7 +137,7 @@
         <RelativeLayout
             android:id="@+id/frame_today_icon"
             android:layout_width="wrap_content"
-            android:layout_height="match_parent"
+            android:layout_height="0dp"
```
Comment (jjchan1) [inline collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml:140]: What is this supposed to do?

File: collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml (hunk 1 above)
Comment (byersc1122) [reply collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml:115]: 0dp with the constraints it has makes the button height as tall as the parent view

File: collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml (hunk 2 above)
Comment (byersc1122) [reply collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml:140]: 0dp with the constraints it has makes the button height as tall as the parent view

**Review by jjchan1 — APPROVED** [review]
LGTM

**Pull request merged**